src/elements/netspeak-search.tsx

```diff
@@ -1,28 +1,29 @@
 import React from "react";
-import { LocalizableProps, Locales, SimpleLocale, createLocalizer } from "../lib/localize";
+import { createLocalizer, Locales, LocalizableProps, SimpleLocale } from "../lib/localize";
 import {
-	Phrase,
-	normalizeQuery,
 	Netspeak,
-	NetspeakInvalidQueryError,
+	NetspeakApi,
 	NetspeakError,
+	NetspeakInvalidQueryError,
+	normalizeQuery,
+	Phrase,
 	ReadonlyNetspeakSearchResult,
 } from "../lib/netspeak";
```

Comment:
It's a good idea to sort these, but feature and stylistic changes should be separate PRs.

Let's include the stylistic changes for now. I'll make a PR for string imports after merging this one.

Reply:
Never heard of that being used in webdev but it works I guess?